### ISSUE TITLE
Ensuring execs trust AI systems

### ISSUE THREAD

**Issue by adamboazbecker** [mlopscommunity/open-questions-ai-quality#12]
What are some strategies to help ensure company execs trust our AI systems?

**Comment by brian386** [comment]
Two main things I can think of are:
1) Thinking of a good set of metrics that capture desired behaviour of the AI model. A model that is "bad" in some way should get penalized by at least one of our metrics.
2) Explaining the metric in a way that's understandable to the execs. 
If we can create a set of metrics that are comprehensive, and we build a AI system that satisfy these metrics, then we can gain trust from clients. 

Also, open-sourcing and reproducability is important towards trust. Allowing others to reproduce experiments will convince them that we're not doing anything wrong under the hood.

**Comment by EliottZemour** [comment]
I completely agree with Brian. I would also add that building _detailed_, _comprehensive_, and _audience-adapted_ documentation that can be shared across teams is important. Some examples I can think of are:
- Model cards displaying key metrics, as well as details regarding compliance, data and compute. 
- Red Teaming reports to detail and summarize findings of Vulnerability assessments.
- Performance reports, including edge-testing and stress-testing

**Comment by pushkargarg** [comment]
1. Finding ways of measuring impact of AI systems.
2. Explainability - revealing the most important features for each prediction
3. Enabling fallback and adding guardrails to hallucination type predictions.
